<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDashboard</class>
 <widget class="QMainWindow" name="AdminDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员控制台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_flight">
       <attribute name="title">
        <string>航班管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <!-- START: 航班搜索布局 -->
        <item>
         <widget class="QGroupBox" name="groupBox_Search">
          <property name="title">
           <string>航班搜索</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>出发地:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtSearchOrigin"/>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>目的地:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtSearchDestination"/>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>日期:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dtSearchDate">
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSearchFlight">
             <property name="text">
              <string>搜索</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <!-- END: 航班搜索布局 -->
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnAddFlight">
            <property name="text">
             <string>添加航班</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditFlight">
            <property name="text">
             <string>修改航班</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDeleteFlight">
            <property name="text">
             <string>删除航班</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnRefresh">
            <property name="text">
             <string>刷新数据</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="flightTable">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <!-- START: 航班分页布局 -->
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnPrevPage">
            <property name="text">
             <string>上一页</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblPageInfo">
            <property name="text">
             <string>Page 0/0 (0 条结果)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnNextPage">
            <property name="text">
             <string>下一页</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <!-- END: 航班分页布局 -->
       </layout>
      </widget>
      <widget class="QWidget" name="tab_user">
       <attribute name="title">
        <string>用户管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="userTable">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_booking">
       <attribute name="title">
        <string>订单管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <!-- START: 订单搜索布局 (新增) -->
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>用户ID搜索:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtSearchUserID"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnSearchUserOrders">
            <property name="text">
             <string>查询用户订单</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <!-- END: 订单搜索布局 -->
        <item>
         <widget class="QTableWidget" name="bookingTable">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
